waddrmgr/db.go

```diff
@@ -1866,6 +1870,17 @@ func putSyncedTo(ns walletdb.ReadWriteBucket, bs *BlockStamp) error {
 	if err != nil {
 		return managerError(ErrDatabase, errStr, err)
 	}
+
+	// Finally we delete any hashes for heights below our max reorg depth.
+	forgetHeight := bs.Height - maxReorgDepth
```

Comment:
Just so we can be sure of the behavior here, can we cast this to an int32?

Reply:
`bs.Height` already is, so it will be `int32`.

Reply:
👍